_schemaV2.graphql

```diff
@@ -10941,6 +10947,12 @@ type Viewer {
     width: String
   ): FilterArtworksConnection
 
+  # An auction result
+  auctionResult(
+    # The ID of the auction result
+    id: String!
+  ): AuctionResult
+
```

Comment:
Should this be here twice?

Reply:
yeah. if memory serves we have this `viewer` root field which is just a duplication of the top-level `Query` type because it is/(used to be?) awkward to do certain kinds of query in relay on the top level.